server/lib/taxForms.js

```diff
@@ -0,0 +1,88 @@
+import models from '../models';
+import { uniqBy } from 'lodash';
+const { RequiredLegalDocument, LegalDocument, Collective, User } = models;
+const {
+  documentType: { US_TAX_FORM },
+} = RequiredLegalDocument;
+
+export async function findUsersThatNeedToBeSentTaxForm({ invoiceTotalThreshold, year }) {
```

Comment:
An elegant way of checking that could be to have a `LEFT OUTER JOIN` between `RequiredLegalDocuments` and `LegalDocuments`, with a JOIN condition on year/user and `requestStatus != RECEIVED`

Reply:
I've spend some time looking at this and I can't see how this would work. We need to get the user total expenses as well.

Also, this query is only used by the cron job so it's not in a hot code path, so can we just leave it as is?

Reply:
Totally, I missed the fact that this was only used by the CRON and I'm really fine with it.

GitHub should let you precise on individual review comments if they are just thought or actual blockers :)